strawberry/types/types.py

```diff
@@ -113,6 +114,16 @@ def fields(self) -> List[StrawberryField]:
     def is_generic(self) -> bool:
         return is_type_generic(self.origin)
 
+    @property
+    def is_generic_specialized(self) -> bool:
```

Comment:
```suggestion
    def is_specialized_generic(self) -> bool:
```

I think this sounds better 😊